content/src/Blip.Api.Template/Models/NLPContentProviderModels/ContentResultStatus.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Blip.Api.Template.Models.NLPContentProviderModels
```

Comment:
This should be removed alonside any mention of ContentProviderAPI, and moved to an independent package